src/Test.UnitTests.Sarif.Multitool/ResultMatchingCommandTests.cs

```diff
@@ -0,0 +1,172 @@
+﻿using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Baseline.ResultMatching;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
+{
+    public class ResultMatchingCommandTests
```

Comment:
>ResultMatchingCommandTests [](start = 17, length = 26)

This is a good start. Some ideas: first, make sure and use (and build out if required) our standard test helpers for creating testable log files, see src\Test.Utilities.Sarif\TestData.cs. e.g., CreateSimpleLog(). Next, build out these tests to cover all possible states, absent, new, unchanged, etc. Minimal tests are fine. Finally, be sure to create a test that explicitly covers the bug you've fixed. Don't use any real world MS repro content for this, instead, check in constructed files that provoke the issue. 

btw - are these tests in the right place? looking at the content, this test files s/be testing the high level match commands, but instead, they are testing the lower level functionality. Did you look at the content in src\sarif\src\Test.UnitTests.Sarif\Baseline\ResultMatching? These tests likely already have basics of new, unchanged, etc. So maybe we can drop this file and go add new cases to that other content?

Reply:
Just added more sarifs in TestData and moved the test itself to the same file we were using for baselining.

---
In reply to: [502492851](https://github.com/microsoft/sarif-sdk/pull/2102#discussion_r502492851) [](ancestors = 502492851)